packages/client/src/features/tools/change-bounds-tool.ts

```diff
@@ -246,9 +244,6 @@ export class ChangeBoundsListener extends DragAwareMouseListener implements Sele
                 element.outgoingEdges.map(toElementAndRoutingPoints).forEach(ear => newRoutingPoints.push(ear));
             }
         });
-        if (newRoutingPoints.length > 0) {
-            result.push(ChangeRoutingPointsOperation.create(newRoutingPoints));
```

Comment:
Removing this introduces a regression bug.
If an node is moved and the connected edges a rerouted in the process the changed routing points are no longer sent to  the server. As a consequence an new route based on the old routingpoints is computed after the server update.
(This bug is not visible with polyline routing, you would have to use an manhattan edge to test this)

However, we should probably reuse the update method that you introduced with this PR.

Reply:
Good catch! I extracted the calculation of the routing points and now share the functionality between both tools. Thank you!